### PR TITLE
docs: add pull request template

### LINKED ISSUE #1596: docs: add PR template
When you add a pull request template to your repository, project contributors will automatically see the template's contents in the pull request body.

For more information read this documentation -> https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

### PR DESCRIPTION
**Pull request by YashPimple**
fixes #1596 

Here comes the changes proposed through this PR:
- [X] Created pull_request.md file as per instructed 

Signed-off-by: Yash Pimple <yashpimple22@gmail.com>

### CODE REVIEW

**Review by DavidPHirsch — APPROVED** [review]

**Review by mowies — CHANGES_REQUESTED** [review]
please put the template into the `PULL_REQUEST_TEMPLATE` folder next to the issue template folder

**Comment by YashPimple** [comment]
@mowies I have made the required changes. Please review and let me know if any further adjustments are needed

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<!--Type in all the issues that have been fixed through this pull request ex : #1 -->
+
+## Fixes Issue
+
+This PR fixes the following issues:
```
Comment (mowies) [inline .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:5]:
```suggestion
Fixes #XXXX <!-- replace with the issue number that is fixed in this PR -->
```

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+#example
```
Comment (mowies) [inline .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:7]:
```suggestion

```

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+<!-- Write down all the changes made-->
+## Changes proposed
+
+Here comes all the changes proposed through this PR
+
+<!-- Check all the boxes which are applicable to check the box correct follow the following conventions-->
+<!--
+[x] - Correct
+[X] - Correct
+-->
```
Comment (mowies) [inline .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:18]: I don't think we need checkboxes for this necessarily
```suggestion
- change one
- change two
- ...
```

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+## Check List (Check all the boxes which are applicable) <!--Follow the above conventions to check the box-->
```
Comment (mowies) [inline .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:20]:
```suggestion
## Check List (Check all the boxes that are applicable)
```

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+- [ ] My code follows the code style of this project.
+- [ ] My change requires a change to the documentation.
+- [ ] I have updated the documentation accordingly.
```
Comment (mowies) [inline .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:24]: This repo is the documentation. I don't think we need this here.

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+- [ ] All new and existing tests passed.
+- [ ] This PR does not contain plagiarized content.
+- [ ] The title of my pull request is a short description of the requested changes.
```
Comment (mowies) [inline .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:27]:
```suggestion
- [ ] The title of my pull request is a short description of the requested changes.
- [ ] The PR title follows semantic commit conventions.
```

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+
+<!--Add screen shots of the changed output-->
+## Screenshots 
+Add all the screenshots which support your changes
```
Comment (mowies) [inline .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:32]:
```suggestion

```

**Review by mowies — CHANGES_REQUESTED** [review]
I added some things to change. I think we can make things a bit simpler here

**Comment by YashPimple** [comment]
@mowies I have made the required changes. Please do have a look

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md [hunk 8]
```diff
@@ -11,22 +11,10 @@ This PR fixes the following issues:
...
+- ...
 
-## Check List (Check all the boxes which are applicable) <!--Follow the above conventions to check the box-->
-
-- [ ] My code follows the code style of this project.
-- [ ] My change requires a change to the documentation.
-- [ ] I have updated the documentation accordingly.
-- [ ] All new and existing tests passed.
-- [ ] This PR does not contain plagiarized content.
+## Check List (Check all the boxes that are applicable)
```
Comment (mowies) [inline .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:18]: i didn't mean that you need to remove the whole list here. just the lines i mentioned :)

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md [hunk 9]
```diff
@@ -2,7 +2,7 @@
 
 ## Fixes Issue
 
-This PR fixes the following issues:
+Fixes #XXXX <!-- replace with the issue number that is fixed in this PR -->
 
 #example
```
Comment (mowies) [inline .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:7]: pls remove this line

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md (hunk 8 above)
Comment (YashPimple) [reply .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:18]: Got it, I'll remove the lines you mentioned and leave the rest of the list as is. Thanks for the heads up!

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+## Fixes Issue
+
+Fixes #XXXX <!-- replace with the issue number that is fixed in this PR -->
+
+
+<!-- Write down all the changes made-->
+## Changes proposed
+
+Here comes all the changes proposed through this PR
```
Comment (mowies) [inline .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:11]:
```suggestion
## Proposed changes
```

**Review by mowies — APPROVED** [review]

**Comment by mowies** [comment]
@YashPimple LGTM!
Before merging you need to add sign-off to all your commits in this PR. check the DCO check [here](https://github.com/keptn/keptn.github.io/pull/1602/checks) on how to do it if you don't know.

**Review by mowies — CHANGES_REQUESTED** [review]
Will leave this on "request changes" until you have signed all the commits :)

**Comment by aroramrinaal** [comment]
@bhavyastar @mowies , is this issue still open?

**Comment by StackScribe** [comment]
Keptn V1 has reached EOL on December 22nd, 2023, so this PR has been closed.

**Pull request closed without merging**